project-fortis-spark/src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventsSink.scala

```diff
@@ -83,43 +80,28 @@ object CassandraEventsSink {
         consistencyLevel = ConsistencyLevel.ALL // Ensure write is consistent across all replicas
```

Comment:
Would local quorum consistency also be good enough here? We're now using local quorum for reads where previously we were using local one, so we should be able to get consistent read behavior with lower write consistency.

Reply:
Local quorum it is!